plugin.rb

```diff
@@ -104,6 +104,20 @@ class Engine < ::Rails::Engine
 
     reaction_users_counting_as_like.flatten!
 
+    # Used to remove historical like data in the filter below.
+    previous_main_reaction_like_ids =
```

Comment:
We can potentially add a spec using `track_sql_queries` for this, since this is one of many `add_to_serializer` modifications to the `PostSerializer`, it's not really possible to do a clean before/after comparison without some more extensive refactoring, which I do not think is worth it at this time. Confirmed that the N1 is now gone locally.